.coderabbit.yaml

```
@@ -9,7 +9,7 @@ reviews:
   high_level_summary_placeholder: '@coderabbitai summary'
   auto_title_placeholder: '@coderabbitai'
   auto_title_instructions: ''
-  review_status: false
+  review_status: true 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)